pkg/oomstore/apply.go

```diff
@@ -165,7 +168,7 @@ func (s *OomStore) applyFeature(ctx context.Context, txStore metadata.WriteStore
 	return nil
 }
 
-func buildApplyStage(ctx context.Context, opt apply.ApplyOpt) (*apply.ApplyStage, error) {
+func BuildApplyStage(ctx context.Context, opt apply.ApplyOpt) (*apply.ApplyStage, error) {
```

Comment:
@lianxmfor fyi

Reply:
The reason `buildApplyStage` is private is that I don't want to expose the oomstore layer to the public, but being private would also result in the test files having to be under this package as well.
If you feel comfortable with the oomstore layer exposing x to the public, that's fine

Reply:
discussed offline, we decided to keep the function private